pkg/plugin/v3/webhook.go

```diff
@@ -106,9 +113,21 @@ func (p *createWebhookPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
 		return nil, fmt.Errorf("unable to load boilerplate: %v", err)
 	}
 
+	// Load the requested plugins
+	plugins := make([]model.Plugin, 0)
+	switch strings.ToLower(p.pattern) {
+	case "":
+		// Default pattern
+	case AddonPlugin:
+		plugins = append(plugins, &addon.Plugin{})
+	default:
+		return nil, fmt.Errorf("unknown pattern %q", p.pattern)
+	}
+
```

Comment:
Why do we need to add this code?

Reply:
I missed the train of thoughts here. Added this to have a addon plugins supported for the webhook. Just same as the api.go has. Let me remove all the out of scope code from this PR. Thanks for pointing it out.

Reply:
Removed it.